content/applications/finance/payment_providers/demo.rst

```diff
@@ -0,0 +1,129 @@
+=====================
+Demo payment provider
+=====================
+
+Odoo's **Demo Payment Provider** allows you to test business flows involving online transactions
+without requiring real banking credentials.
+
+.. seealso::
+   :doc:`../payment_providers`
+
+Installation
+------------
```

Comment:
Even if it works, we made all headings markups consistent in this repository, please use the same ones as described here
https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#headings